modules/os2forms_webform_list/os2forms_webform_list.info.yml

```diff
@@ -1,9 +1,9 @@
 name: OS2forms webform list
 type: module
-description: Alters display of webform list and maestro template list
+description: Alters display of webform list (/admin/structure/webform)
 package: OS2Web
 core: 8.x
 core_version_requirement: ^8 || ^9
 dependencies:
   - 'drupal:webform'
-  - 'drupal:maestro'
+  - 'drupal:os2forms_permissions_by_term'
```

Comment:
I think this should be `os2forms_permissions_by_term:os2forms_permissions_by_term` since `os2forms_permissions_by_term` is a custom module
(cf. https://www.drupal.org/docs/develop/creating-modules/let-drupal-know-about-your-module-with-an-infoyml-file#s-complete-example), but I don't know if it actually makes any difference. Hopefully it doesn't.

Reply:
Fixed